modules/simple_forms_api/app/services/simple_forms_api/form_remediation/file_utilities.rb

```diff
@@ -78,11 +78,11 @@ def write_file(dir_path, file_name, content)
         File.write(File.join(dir_path, file_name), content)
       end
 
-      def unique_file_name(form_number, id, date = Time.zone.today)
+      def unique_file_name(form_number, id, date = Time.now.utc.to_date)
```

Comment:
Would `Time.zone.now` work? Or do we need to specify `utc` because of AWS?

Reply:
The line of thinking is that we need to specify a standard timezone to ensure that both the creation and the querying of a given submission will always return the correct date.